Documentation/bpf.rst

```diff
@@ -1181,6 +1181,64 @@ The generated LLVM IR can also be dumped in human readable format through:
 
     $ clang -O2 -Wall -emit-llvm -S -c xdp-example.c -o -
 
+In addition, For conversion debugging information to BTF (BPF Type Format),
+more recent ``llc`` and ``pahole`` are needed and ``llc`` and ``pahole``
+can be probed for BTF support:
+
+::
+
+    $ llc -march=bpf -mattr=help |& grep dwarfris
+      dwarfris - Disable MCAsmInfo DwarfUsesRelocationsAcrossSections.
+      dwarfris - Disable MCAsmInfo DwarfUsesRelocationsAcrossSections.
+      dwarfris - Disable MCAsmInfo DwarfUsesRelocationsAcrossSections.
+    $ pahole --help | greo BTF
+    -J, --btf_encode           Encode as BTF
+
+Recent pahole version can be gotten from https://git.kernel.org/pub/scm/devel/pahole/pahole.git
+and note that the ``llvm-objcopy`` tool is required for ``pahole``, so probe it, too.
+
+``-mattr=dwarfris`` should be used with llc and enables ``pahole`` to dump structures correctly.
+(For ``pahole``, dwarf cross-section relation should be disabled through ``-mattr=dwarfris``,
+because BPF backend is not yet implemented in elfutils backend)
+
+Like the above two steps, generating a binary LLVM IR intermediate file
+``xdp-example.bc`` with debugging informantion by adding ``-g``, which
+can later on be passed to llc with ``-mattr=dwarfris``:
+
```

Comment:
Could you also add an example for using clang only? That should be: `clang -target bpf -O2 -g -c -Xclang -target-feature -Xclang +dwarfris t.c` See also: https://reviews.llvm.org/rL334839

Reply:
Yep!

Reply:
Thanks @Taeung I will take a look at the above clang/llvm error as soon as possible.